nginx/nginx.conf

```
@@ -0,0 +1,18 @@
+events {}
+
+http {
+    server {
+        listen 5001;
+        listen [::]:5001;
+
+        server_name _;
+
+        location /api/ {
+            proxy_pass http://backend:5000;
+        }
+
+        location / {
+            proxy_pass http://frontend:6666;
```

I've updated #56 so that the frontend is using 5001, as 6666 was causing trouble.
```suggestion
            proxy_pass http://frontend:5001;
```